src/Cryptol/REPL/Command.hs

```diff
@@ -183,6 +184,10 @@ nbCommandList  =
     "do type specialization on a closed expression"
   , CommandDescr [ ":eval" ] (ExprArg refEvalCmd)
     "evaluate an expression with the reference evaluator"
+  , CommandDescr [ ":ast" ] (ExprArg astOfCmd)
+    "print out an AST of a given term"
+  , CommandDescr [ ":all" ] (ExprArg allTerms)
```

Comment:
* ast-all is great
* I didn't fully understand ExprArg when I wrote this. If there's a more appropriate alternative I'm happy with it.